File: PRAK505-2510817220023-NURFITRIA RAMADHANI.py
def Biodata(tahunLahir,Namaku,Asal):
    tahun_sekarang=2020
    print(f"Perkenalkan Nama Saya : {Namaku}")
    print(f"Umur Saya : {tahun_sekarang-tahunLahir}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {Asal}")
    
tahunLahir=int(input())
A=input()
B=input()
Biodata(tahunLahir,A,B)

File: PRAK505-2510817220023-NURFITRIA RAMADHANI.c
#include <stdio.h>
void Biodata(int tahunLahir,char Namaku[20],char Asal[15]){
int tahun_sekarang = 2020;
printf("Perkenalkan Nama Saya : %s\n", Namaku);
printf("Umur Saya : %d\n", tahun_sekarang-tahunLahir);
printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
printf("Asal Saya dari : %s\n", Asal);
}
int main() {
int tahunLahir;
char A[20], B[15];
scanf(" %d",&tahunLahir); 
scanf(" %[^\n]%*c",&A);
scanf(" %[^\n]%*c",&B); 
Biodata(tahunLahir,A,B); 
return 0;

}
